modules/sdk/gradle-plugins-workspace/src/gradleTest/targetPlatformVersion/modules/foo-portlet/build.gradle

```diff
@@ -1,6 +1,4 @@
 dependencies {
-	compile group: "com.google.guava", name: "guava", version: "23.0"
```

Comment:
@gamerson, aren't we using Guava in the portlet?

Reply:
We were, but I removed it, as it isn't necessary for this test, I had just copied it from another test.